src/async_impl/body.rs

```diff
@@ -137,8 +137,24 @@ impl Body {
         }
     }
 
-    // pub?
-    pub(crate) fn streaming<B>(inner: B) -> Body
+    /// Wrap a `HttpBody` in a box inside `Body`.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// # use reqwest::Body;
+    /// # use futures_util;
+    /// # fn main() {
+    /// let content = "hello,world!".to_string();
+    ///
+    /// let body = Body::streaming(content);
+    /// # }
+    /// ```
+    ///
+    /// # Optional
+    ///
+    /// This requires the `stream` feature to be enabled.
+    pub fn streaming<B>(inner: B) -> Body
```

Comment:
I'm not sure if it's ok to keep the naming. I'm open to other candidates. cc @seanmonstar

Reply:
Hm, maybe `Body::wrap(other)`? Since a particular implementation might not _actually_ be streaming?

Reply:
DONE. @seanmonstar, please take a look.